sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -555,10 +574,12 @@ func componentsToResourceData(components []*cloudauth.AccountComponent) []map[st
 
 // internal type redefintion for GCP service principals.
 // This exists because in terraform, the key is originally provided in the form of a base64 encoded json string
+
+// note; caution with order of fields, they have to go in alphabetical ASC so that the json marshalled on the tf read phase produces no drift https://github.com/golang/go/issues/27179
```

Comment:
cc @cgeers just so you are aware of this.
for me it's ok to merge with this

Reply:
haha, yeah...  if you want to dig more into the issue feel free to do so, but let's agree on some criteria to accept this PR and unblock us;  you can work on digging more  anytime later.

i was there for couple of days and i'm ok letting it go... 
gotta say though, i'm not expert on go, neither the json libraries. maybe you do see some other option